### PR TITLE
Refactoring of emails settings

### PR DESCRIPTION
**Pull request by kojoty**
This is simple refactor - I made little cleanup around emails addresses.

@harrieklomp @andrixnet 
please check if new configs for your nodes are OK

@deg-pl 
I need config for OCUK - can you prepare such file or send me current setting.inc.php from OCUK?

### CODE REVIEW

**Comment by harrieklomp** [comment]
Done for OCNL

**Comment by deg-pl** [comment]
@kojoty - done

File: okapi_settings.php [hunk 1]
```diff
@@ -37,26 +47,41 @@ function get_okapi_settings()
...
+
+        'FROM_FIELD' => OcConfig::getEmailAddrNoReply(),
         'DEBUG' => $debug_page,
         'DB_SERVER' => $dbserver,
         'DB_NAME' => $dbname,
         'DB_USERNAME' => $dbusername,
         'DB_PASSWORD' => $dbpasswd,
-        'SITELANG' => 'en', //TODO: how to read it from I18n class?
-        'SITELANGS' => array_map('strtolower', $config['defaultLanguageList']),
+        'SITELANG' => I18n::getDefaultLang(),
```
Comment (kojoty) [inline okapi_settings.php]: @following5 Please take a look at this - this is my proposal of the solution for access to OCPL classes (OcConfig) to read new settings - what do you think?

I call OCPL autoloader and next restore OKAPI settings...

Comment (following5) [reply okapi_settings.php]: I would rather avoid to call I18n and OcConfig code from OKAPI for performance reasons. OcConfig creates ApplicationContainer, ApplicationContainer creates database connection ... and whatever else may happen there.

Just working on another proposal which avoids that and is very extensible with null performance penalty.

Comment (following5) [reply okapi_settings.php]: But I am fine with your above proposal as temporary workaround. Can remove the language workaround then from OKAPI code.

My proposal needs the UpdateController, whch - I guess - is not installed yet in OC PL and UK deployment scripts?

Comment (following5) [reply okapi_settings.php]: ... temporary solution, not workaround. It will do it, just can be optimized in performance. :)

Comment (following5) [reply okapi_settings.php]: @kojoty
OKAPI will need translations of country names etc. Currently quickly to fetch from `countries` table, but soon will require loading bulky OCPL translation files!

**Comment by kojoty** [comment]
OK, so now here we have:
- simple OCConfig class (without any external depredations)
- simple okapi_settings - just calls to OCConfig

@following5 
can we merge this PR now?

**Comment by following5** [comment]
I get this error when testing OKAPI:

`Fatal error: okapi\get_okapi_settings(): Failed opening required '/srv/ocpl/lib/settingsGlue.inc.php' (include_path='.:/usr/share/php') in /srv/ocpl/okapi_settings.php on line 30`

**Comment by following5** [comment]
There was some very stange problem with Git checkout on my host system. Ok after reboot.

Thumbs-up for the okapi_settings change.

**Pull request merged**